bpf/lib/eth.h

```diff
@@ -109,6 +109,21 @@ static __always_inline int eth_store_daddr(struct __ctx_buff *ctx,
 #endif
 }
 
+static __always_inline int eth_memcpy_daddr(struct __ctx_buff *ctx,
+                                          const __u8 *mac, int off)
+{
+	void *data_end = ctx_data_end(ctx);
+	void *data = ctx_data(ctx);
+
+	if (ctx_no_room(data + off + ETH_ALEN, data_end))
+		return -EFAULT;
+	/* Need to use builtin here since mac came potentially from
+	 * struct bpf_fib_lookup where it's not aligned on stack. :(
```

Comment:
Can you add some details to the commit message. When does this come from the bpf_fib_lookup()? Is it in the nodeport case?

Reply:
Sorry the "bpf_fib_lookup()" comment is copy and paste from eth_store_daddr(), I shoud remove it.